modules/text2vec-openai/module_test.go

```diff
@@ -0,0 +1,110 @@
+package modopenai
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/semi-technologies/weaviate/entities/moduletools"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
```

Comment:
Is there a reason why are you using this library instead of `"github.com/stretchr/testify/assert"` which is already included?

Reply:
I fixed it and squashed commits in a single commit and the `go mod tidy` commit is no longer needed now.